src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -1119,6 +1124,15 @@ public OpenLoadFlowParameters setWriteReferenceTerminals(boolean writeReferenceT
         return this;
     }
 
+    public List<String> getVoltageTargetPriorities() {
+        return voltageTargetPriorities;
+    }
+
+    public OpenLoadFlowParameters setVoltageTargetPriorities(List<String> voltageTargetPriorities) {
```

Comment:
Defining the parameters in a programmating way does not check that the strings are part of the possible enum values. We already have the issue for outerloop names indeed. Do we need a check?

Reply:
added checks incl. tests, much better this way indeed